### PR TITLE
General Styling Update

### PR DESCRIPTION
**Pull request by notnazsty**
Fixed the major merge issues so it doesn't break 

- Mobile View 
     - Navbar
     - List Page
- Map Filtering
- Map Styling
- List Page Styling

### CODE REVIEW

File: components/Layout/Navbar.tsx [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
+  useDisclosure,
+  VStack
 } from '@chakra-ui/react'
+import React from 'react'
 
 export const Navbar = (): JSX.Element => {
+
+
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const btnRef = React.useRef()
```
Comment (dangalan) [inline components/Layout/Navbar.tsx:26]: This looks unused? So you can delete it and the `React` import

File: components/ServiceItem.tsx [hunk 2]
```diff
@@ -22,56 +22,58 @@ export const ServiceItem = ({
...
-              <Text
-                key={i}
-                bgColor="lightPink"
-                borderRadius="8px"
-                p="8px"
-              >
-                {taxonomy}
-              </Text>
-            ))}
+      <Box boxShadow='md' rounded='lg' p='8' _hover={{ border: '2px', borderColor: 'teal' }} >
```
Comment (dangalan) [inline components/ServiceItem.tsx:31]: You can just pass the styling props to the top level `Box` component after the spread props `{...props}` instead having two `Box` components

File: pages/index.tsx [hunk 3]
```diff
@@ -20,10 +22,58 @@ export default function Home(): JSX.Element {
...
   const { taxonomyOptions, setFilters } = useTaxonomyFilter(baseServicesLists, setServicesLists, filterFunction)
 
-  const sortFieldsTextToVal = {Name: 'name', Description: 'description'}
+  const sortFieldsTextToVal = { Name: 'name', Description: 'description' }
 
   return (
-    <>
+    <Box w='100%' mb={16} display='flex' flexDirection='column'>
+
+      <Heading fontSize='5xl' textAlign={{ base: 'center', md: 'center' }} mt={8}>Services Lists</Heading>
```
Comment (dangalan) [inline pages/index.tsx:30]: I feel like centering this heading looks weird?
Screen Shot 2022-01-17 at 9 20 16 PM

File: pages/index.tsx [hunk 4]
```diff
@@ -20,10 +22,58 @@ export default function Home(): JSX.Element {
...
+                  primary25: '#B2DFDB',
+                  primary: 'black',
+                },
+              })}
+            />
+          </Box>
+
+        </HStack>
+      </VStack>
+      <Stack spacing='36px' alignItems='left' pl={{ base: 4, sm: 16, md: 32 }} >
```
Comment (dangalan) [inline pages/index.tsx:58]: Adding padding only on the left side here also looks weird

File: components/Layout/Navbar.tsx [hunk 5]
```diff
@@ -1,24 +1,41 @@
...
+    <Flex bgColor='darkTeal' color='white' justifyContent='space-between' alignItems='center' >
+      <Image
+        src='/manyc_logo_no_background.png'
+        w='108px'
+        h='96px'
+        alt="Mutual Aid NYC"
+      />
+      <HStack spacing='32px' pr={4} display={{ base: 'none', md: 'inherit' }}>
+        <Link>
+          Services
```
Comment (dangalan) [inline components/Layout/Navbar.tsx:38]: `href` for this link and the logo got lost in the merge I think

File: components/Layout/Navbar.tsx [hunk 6]
```diff
@@ -30,6 +47,46 @@ export const Navbar = (): JSX.Element => {
...
+                  Lists
+                </Link>
+                <Link>
+                  Submit A New List
+                </Link>
+                <Link>
+                  Organizations
+                </Link>
+                <Link>
+                  About
```
Comment (dangalan) [inline components/Layout/Navbar.tsx:82]: Need `href` values for all these links

File: pages/index.tsx (hunk 3 above)
Comment (notnazsty) [reply pages/index.tsx:30]: Oh yeah I think it's because I messed up the merge for the page

**Review by btmoore4 — APPROVED** [review]

**Comment by notnazsty** [comment]
Had an issue loading certain modules for some reason but installing "ts-loader": "^9.2.6" seemed to fix it. That was the major issue with the yarn.lock file. Let me know if it creates any issues on your end.

**Pull request merged**